CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   post script instance.
 - Add `dnf` definition file bootstrap options as an alias to `yum`.
 - Add --sandbox option to apptainer pull.
+- Add configuration files binding for `--nv` option.
```

Comment:
Also have an entry about the additional files added to nvliblist.conf, e.g.:
```
- Add the confliguration files to nvliblist.conf that are recognized in the NVIDIA Container Toolkit, including files for EGL ICD.
```

Reply:
I revised the changelog entry for clarity.